fluidBuild.config.cjs

```diff
@@ -60,6 +60,8 @@ module.exports = {
 			"api-extractor:commonjs",
 			"api-extractor:esnext",
 		],
+		"build:test:cjs": ["typetests:gen", "tsc", "api-extractor:commonjs"],
+		"build:test:esm": ["typetests:gen", "build:esnext", "api-extractor:esnext"],
```

Comment:
@tylerbutler, if a package doesn't have a `tsc` or `build:exnext` script would the general dependencies of `tsc` on `^tsc` and `build:esnext` on `^build:esnext` be transitively picked up? Or would they get ignored?

Reply:
They'd be ignored. The dependency "chain" is broken in that case so unless another task brought those tasks in, they'd be omitted. 

@curtisman did I get that right?

Reply:
Yes, the relevent code is in `buildGraph.ts`: `BuildPackage.getMatchedTasks`